pkgs/tools/misc/byobu/default.nix

```diff
@@ -1,5 +1,8 @@
-{ stdenv, fetchurl, python3, perl, textual-window-manager }:
+{ stdenv, fetchurl, python3, perl, textual-window-manager, screen, newt, makeWrapper }:
 
+let
+  newtPythonLib = "${newt}/lib/python3/site-packages";
+in
```

Comment:
This is quite ugly, surely there is a better way to do this?

Reply:
`${python.sitePackages}`